docs/index.html

```diff
@@ -98,10 +121,34 @@ <h2 id="setup">Setup</h2>
 cd &quot;$GOPATH/src/github.com/u-root/u-root&quot;
 </code></pre>
 
-<p>Test u-root inside a chroot:</p>
-<pre><code class="sh">go run scripts/ramfs.go -test
+<p>Generate an initramfs of all u-root Go tools:</p>
+<pre><code class="sh">go run u-root.go -o initramfs.cpio
+</code></pre>
+
+<p>You can use this initramfs with your favorite Linux kernel in QEMU to try it
+out.</p>
+<p>More instructions can be found in the repo's
+<a href="https://github.com/u-root/u-root/blob/master/README.md">README.md</a>.</p>
+<h2 id="using-elvish-a-more-handy-shell">Using elvish: a more handy shell</h2>
+<p>By default, the u-root shell is rush. However, rush does not have very many
+features.</p>
+<p>Now, thanks to Qi Xiao (<a href="&#109;&#97;&#105;&#108;&#116;&#111;&#58;&#120;&#105;&#97;&#113;&#113;&#97;&#105;&#120;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;">&#120;&#105;&#97;&#113;&#113;&#97;&#105;&#120;&#64;&#103;&#109;&#97;&#105;&#108;&#46;&#99;&#111;&#109;</a>), u-root users are also able to
```

Comment:
I am wondering about removing references to elvish given what a massive security hole direction it is taking. I don't know what to do.

Reply:
Removed it. The readme has a reference to it anyway.